react-app/src/pages/LoginPage.js

```diff
@@ -29,10 +31,14 @@ function LoginPage({ logUserIn }) {
 
     function onAvatarImgClick(user) {
         if (user.role === ADMIN_USER) {
-            history.push('/secure-login')
+            setAdminUser(user)
         } else {
-            history.replace('/')
+            logUser(user)
         }
+    }
+
+    function logUser(user) {
```

Comment:
I would suggest using `signUserIn` as a name, instead of `logUser`. 

Log by itself is the same log as in `console.log` - on the first view of the PR I didn't spot this and thought it some debug thing.